sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -4946,6 +4953,8 @@ class SQLConf extends Serializable with Logging {
 
   def parquetInferTimestampNTZEnabled: Boolean = getConf(PARQUET_INFER_TIMESTAMP_NTZ_ENABLED)
 
+  def parquetTimestampNTZEnabled: Boolean = getConf(PARQUET_TIMESTAMP_NTZ_ENABLED)
+
```

Comment:
Looks like this sneaked in while rebasing...

Renamed recently here: https://github.com/apache/spark/commit/ae24327e8984e7ee8ba948e3be0b6b2f28df68d0#diff-13c5b65678b327277c68d17910ae93629801af00117a0e3da007afd95b6c6764L4946-R4947

Reply:
Thank you, this has now be addressed